app/models/concerns/pathways_version_four_calculations.rb

```diff
@@ -8,14 +8,25 @@ module PathwaysVersionFourCalculations
   extend ActiveSupport::Concern
 
   included do
+    PHONE_NUMBER_REGEX = /\(?[\d]{3}\)?[\s|-]?[\d]{3}-?[\d]{4}/
     validates_presence_of :entry_date, :hud_assessment_location, :hud_assessment_type, :setting, on: [:create, :update]
+    validates_email_format_of :email_addresses, if: :pathways?
+    validates_format_of :phone_number, with: PHONE_NUMBER_REGEX, message: 'Must be a valid phone number (10 digits).', if: :pathways?
```

Comment:
This should probably allow blanks?  (maybe it does?)

Reply:
Yes, this should allow for blanks. I'll add that to the email and phone validation.